### ISSUE TITLE
Argon2 v0.25.0 does not work on Windows 10

### ISSUE THREAD

**Issue by vansergen** [ranisalt/node-argon2#220]
I can not use `argon2` after updating from `v0.24.1` to `v0.25.0`

### Before creating an issue, please be sure to:
- [x] Checkout to the latest version, **including submodules**
- [x] Try to find an *isolated* way to reproduce the behavior
- [x] Fill in all the blanks in the most specific way you can

### Steps to reproduce
1. Create an empy project
```bash
npm init
```
2. Install `argon2`
```bash
npm install argon2
```
3. Create the file `index.js`
```javascript
const argon = require("argon2");
```
4. Run `index.js`:
```bash
node index
```

### Expected behaviour
No errors

### Actual behaviour
The error:
```bash
internal/modules/cjs/loader.js:1003
  return process.dlopen(module, path.toNamespacedPath(filename));
                 ^

Error: \\?\C:\GitHub\argon2-test\node_modules\argon2\lib\binding\argon2.node is not a valid Win32 application.
```

### Environment
**Operating system**: Windows 10

**Node version:** `12.13.0`

**Argon version:** `0.25.0`

**Comment by vansergen** [comment]
It seems that the error was fixed in [`v0.25.1`](https://github.com/ranisalt/node-argon2/releases/tag/v0.25.1). But it is not published on [npm](https://www.npmjs.com/)

**Comment by ranisalt** [comment]
Published now.
